main.js

```diff
@@ -0,0 +1,67 @@
+let apis;
+let savedUsers = [];
+
+//load data from apis
+$('#data-btn').on('click', function () {
+  apis = new APIManager();
+  apis.getAll();
+  $('#display-btn').removeClass('invisible');
+});
+
+//show loaded user
+$('#display-btn').on('click', function () {
+  const rend = new Renderer();
+  const friends = [];
+  for (i = 1; i < apis.data.userData.length; i++) {
+    friends.push(apis.data.userData[i]);
+  }
```

Comment:
the main.js file shouldn't make "thinking" actions - it just needs to connect between the data to the rendering. You should have do that on your APIManager, saving the friends array in the data variable as a key-value